### ISSUE TITLE
The stop task button appears visibly but is not announced programmatially.

### ISSUE THREAD

**Issue by Tanvez** [Harvard-University-iCommons/chainlit#24]
| IMPACT      | URL          | ISSUE DESCRIPTION   | WCAG CRITERIA | TESTING METHOD | SCREENSHOT or SNIPPET | ACTION |
| :----:          |    :----:      |      :----:                        |  :----:                    | :----:                        | :----:                                     |  :----------------------------------------:    | 
| High    |  n/a      | The stop task button appears visibly but is not announced programmatially.  | 1.3.1 Info and Relationships - A | Manual - Safari/VO | [link](https://share.getcloudapp.com/04uvZRKY) | The "stop task" button is not announced when it appears while the prompt is being processed. <br /> Move focus to the button when it appears so screen reader users know it is available and can use it. When the prompt appears move focus back to the text input.
